src/core/src/bootstrap/Constants.py

```diff
@@ -198,6 +198,7 @@ class StatusTruncationConfig(EnumBackport):
     SUSE = 'SUSE'
     CENTOS = 'CentOS'
     AZURE_LINUX = 'Microsoft Azure Linux'
```

Comment:
Do AZURE_LINUX = ['Microsoft Azure Linux', 'Common Base Linux Mariner'] 

Have a private method in EnvLayer ---

__is_distro_azure_linux (distro)
return any(x in distro for x in Constants.AZURE_LINUX)

Use that in both the files below.

Reply:
Done